Documentation/README_HandCoach.md

```diff
@@ -0,0 +1,104 @@
+# Hand coach
```

Comment:
Would be good to mention that this is experimental so people are aware the component is subject to change.

Also, I noticed we have some inconsistency in how we document experimental features. For example, in [scrolling object collection](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6001/files) we have a readme.md file under MRTK.SDK/Experimental/ScrollingObjectCollection/README.md. But for this feature we are putting it directlyunder `Documentation`. For consistency, how about we agree on either adding documentation under `Documentation/` folder or directly in a README under the experimental folder.

Aside: to increase discoverability and clarity, let's also add an **Experimental** section under **Guides/Feature Overviews**

Reply:
- Moved the README file to Experimental folder
- Updated TOC: Added 'Experimental Features' under Feature Overviews and added links to Experimental features (Hand Coach, Scrolling Object Collection, Pulse Shader)
Thanks!